services/mobile-services-dashboard.adoc

```diff
@@ -0,0 +1,83 @@
+= Mobile Services dashboard in Grafana
+
+== Prerequesites
+
+* OpenShift Origin >= 3.7
+** https://docs.openshift.org/latest/install_config/index.html
+* OpenShift Ansible Broker
+** https://docs.openshift.org/latest/install_config/install/advanced_install.html#configuring-openshift-ansible-broker
+* Metrics service deployment from the Service Catalog
+** https://github.com/aerogearcatalog/metrics-apb
+
+
+== Overview
+
+After the Metrics service (includes Grafana tool and Prometheus monitoring system) is provisioned, 
+you should be able to see the "Mobile Services" in the list of available dashboards 
+(navigate to Grafana's exposed URL -> Log in -> Home -> Select *Mobile Services*)
+
+image:img/mobile-services-dashboard/mobile-services-dashboard-overview.png[mobile-services-dashboard]
+
+The dashboard contains 2 rows with panels, each row for a different service. 
+See the description below for more information about each service's panels.
+
+
+== 1. Keycloak
+
+image:img/mobile-services-dashboard/keycloak-panels.png[keycloak-panels]
+
+=== Keycloak service description
+
+This panel just gives you basic information about the Keycloak service
+
+=== Provisioned
+
+This panel informs you about the status of connection with the Keycloak service.
+
+If you want the Keycloak service's metrics data to be available for Graphana, you first need to deploy it
+from the Service Catalog: https://github.com/aerogearcatalog/keycloak-apb
+
+*Note:* Keycloak service must be provisioned within the same namespace,
+otherwise the Keycloak's metrics endpoint won't be reachable for Prometheus and no data will be displayed.
+
+=== Version
+
+Shows the information about the version of the Node.js
```

Comment:
Ideally this would be the service's version or if this is not possible the **java** version.

Reply:
@psturc Node.js -> Java @josemigallas how do you get the KeyCloak version?

Reply:
@pb82 That's to be figured out yet

Reply:
@josemigallas one option would be to reda the `jave.version` system property in the SPI: https://stackoverflow.com/questions/2591083/getting-java-version-at-runtime

Reply:
Well sure, the Java version is what's currently being displayed. There is actually a metric for that already provided. What I mean is that it would be more useful to now what version of the service itself (APB I guess) is deployed, wouldn't it?